### ISSUE TITLE
Can't enable predictive mode in URI fields

### ISSUE THREAD

**Issue by r0usch** [sspanak/tt9#81]
Video attached. Keyboard is enabled, Eng dictionary loaded, unloaded then reloaded still doesn't display predictive text as a usable mode. Version used in the video is v8.0, though the issue has persisted since v7.0. (I might just be missing something obvious). Phone is a Xiaomi F21 Pro.

https://streamja.com/eERQX (video works best when in fullscreen).

**Comment by sspanak** [comment]
This is probably, because you are trying to use it in the Firefox search bar. Could please try in any other text field? It could even be inside a web page, where it is possible to enter text. Just don't try with this one.

**Comment by sspanak** [comment]
I double checked this.

In some specialized text fields (e.g. password, URL, search, e-mail) Predictive mode is delibaretely forbidden. This logic was implemented by the original author of TT9, many years ago, and I kept it.

As for passwords or emails, I think enabling text predictions is out of the question, because it will lead to more mistakes. However, I see how it can be useful when typing a URL.

Bottom line is: it doesn't bother me, so I do not plan to do it at the moment. But if more people like the idea, I'll do it. Anyone is welcome to give thumbs up to **r0usch**.

**Comment by sspanak** [comment]
That's definitely a valid point.
